Makefile

```diff
@@ -121,6 +121,8 @@ codegen:
 		paths=./...
 	pnpm --dir=ui install --dev
 	pnpm --dir=ui run generate:schema
+	npm install -g @bitnami/readme-generator-for-helm
```

Comment:
I put the install for the readme-generator tool in the target instead of in `ui/package.json` since it's not a part of the UI.